shared/utils.py

```
@@ -414,6 +414,49 @@ def open_folder(path: str) -> Tuple[bool, Optional[str]]:
+            # default library instead of raising or erroring. Quoting only
+            # the path (comma left bare) is the form explorer actually
+            # understands. Safe from injection: `"` is not a legal character
+            # in a Windows path, so norm_path can't break out of the quotes.
+            subprocess.Popen(f'explorer /select,"{norm_path}"')
+        elif system == "Darwin":
+            subprocess.Popen(["open", "-R", norm_path])
+        else:
+            # No universal "select in file manager" command on Linux across
+            # file managers; open the containing directory as the best
+            # available fallback (no highlight).
+            subprocess.Popen(["xdg-open", os.path.dirname(norm_path)])
```

Ruff (0.16.3)

[error] `subprocess` call: check for execution of untrusted input

(S603)